convert.py

```diff
@@ -0,0 +1,108 @@
+from xml.etree import ElementTree as ET
+import json
+import sys
+
+orig_stdout = sys.stdout
+f = open('output.txt', 'w')
+sys.stdout = f
+
+with open("batch_response_data.json") as json_file:
+    json_data = json.load(json_file)
+
+
+ns = {'ns': 'http://www.transunion.com/namespace/pfs/v4'}
+for data in json_data:
```

Comment:
please make a function that takes an xml string as an argument and returns a single rendered report as a list of strings (lines) where each string is exactly 77 characters.